content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -184,7 +184,27 @@ the webhooks. There are three steps to complete the configuration.
   (yes, the same schema that's used by kubectl), so the field name is
   `kubeConfigFile`. Here is an example admission control configuration file:
 
+    {{< tabs name="admissionconfiguration_example1" >}}
+    {{% tab name="apiserver.config.k8s.io/v1" %}}
     ```yaml
+    apiVersion: apiserver.config.k8s.io/v1
+    kind: AdmissionConfiguration
+    plugins:
+    - name: ValidatingAdmissionWebhook
+      configuration:
+        apiVersion: apiserver.config.k8s.io/v1alpha1
```

Comment:
Why alpha here instead of v1?

Reply:
this PR updates the top-level config examples. sub-config updated in https://github.com/kubernetes/website/pull/17549